JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -1,5 +1,25 @@
-The **DateBox** is&nbsp;a&nbsp;widget that displays the date and time in&nbsp;one of&nbsp;DevExtreme [predefined formats][0] or&nbsp;a&nbsp;format defined in&nbsp;an&nbsp;external localization library, or&nbsp;a&nbsp;type in&nbsp;the required date/time value. In&nbsp;this demo, the &laquo;Custom format&raquo; **DateBox**&rsquo;es value is&nbsp;formatted [using the Globalize library][1]. Alternatively, you can [use Intl][2].
+The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) displays date and time in DeveExtreme [predefined formats](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type) or a formats provided by an external localization library such as [Intl](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl). This demo illustrates the following  **DateBox** properties:
```

Comment:
It would be better to mention different formats in the displayFormat description instead.

```suggestion
The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) is an editor that allows users to select date and time and displays them. This demo illustrates the following **DateBox** properties:
```